pages/index.js

```diff
@@ -265,3 +172,96 @@ export default function HomePage({
     </>
   );
 }
+
+const StyledFilterCategoryWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+const FilterWrapper = styled.section`
+  display: flex;
+`;
+
+const ExitButton = styled.button`
+  background-color: rgba(255, 255, 255);
+  position: absolute;
+  top: 0.7rem;
+  right: 0.7rem;
+  border-radius: 5px;
+`;
+
+const FilterButton = styled.button`
+  margin-top: 1rem;
+  background-color: rgba(255, 255, 255);
+  border-radius: 5px;
+  font-size: 0.9rem;
+  margin-left: 5px;
+`;
+
+const StyledFilterForm = styled.form`
+  background-color: rgba(255, 245, 238);
+  border-radius: 8px;
+  padding: 20px;
+  position: relative;
+`;
+
+const StyledRightSide = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledYearInput = styled.input`
+  width: 5em;
+`;
+
+const Overlay = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
```

Comment:
The `inset` property is a shorthand that allows you to set all four of the top, right, bottom, and left properties at once. 

```suggestion
    insert: 0;
```